libs/express/status/vite.config.ts

```diff
@@ -31,7 +32,7 @@ export default defineConfig({
 			// Could also be a dictionary or array of multiple entry points.
 			entry: 'src/index.ts',
 			name: 'express-status',
-			fileName: 'index',
+			fileName: (format) => `express-status.${format}.js`,
 			// Change this to the formats you want to support.
 			// Don't forget to update your package.json as well.
 			formats: ['es', 'cjs', 'umd'],
```

Comment:
Do we still want to provide umd?
Does anyone use it?
I would only provide es and cjs